### PR TITLE
Feature/implement initial active projects view

### PR DESCRIPTION
**Pull request by b1995**
#168

### CODE REVIEW

File: src/pages/ActiveProjectsView.jsx [hunk 1]
```diff
@@ -1,5 +1,30 @@
...
+   ************************************/
+
+  return (
+    <div className="active-projects-view-container">
+      <div className="active-projects-view-body">
+          <div className="active-projects-view-copy-container">
+            <h1>Active Projects</h1>
+            <FeatureContainer image={<ActiveProjectsSVG/>}>
+              <FeatureCopy>
+                Contribute to one of our member-owned projects! Can’t find what your are looking for? Bring in a new project idea or existing project!
```
Comment (Fabricevladimir) [inline src/pages/ActiveProjectsView.jsx:22]: The your in "what your are looking for" should be you.

File: src/pages/ActiveProjectsView.jsx [hunk 2]
```diff
@@ -1,5 +1,30 @@
...
-  return <h1>Active Projects</h1>;
+  /************************************
+   * Render
+   ************************************/
+
+  return (
+    <div className="active-projects-view-container">
+      <div className="active-projects-view-body">
+          <div className="active-projects-view-copy-container">
+            <h1>Active Projects</h1>
```
Comment (Fabricevladimir) [inline src/pages/ActiveProjectsView.jsx:19]: Should that be just Projects instead?

Comment (b1995) [reply src/pages/ActiveProjectsView.jsx]: Should I change the navbar, too? or should that just be a separate ticket?

**Comment by will-nemo** [comment]
> Can we just change every instance of `ActiveProjects` to `Projects` including `active-projects` to `projects`

Could you also change the link label in the toolbar from 'Active Projects to "Projects"

File: src/pages/ProjectsView.jsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  return (
+    <div className="projects-view-container">
+      <div className="projects-view-body">
+        <div className="projects-view-copy-container">
+          <h1>Projects</h1>
+          <FeatureContainer image={<ProjectsSVG />}>
+            <FeatureCopy>
+              Contribute to one of our member-owned projects! Can’t find what you are looking for? Bring in a new project idea or existing project!
+            </FeatureCopy>
+            <Button handleClick={() => window.open(SocialMediaConstants.DISCORD_LINK, "_blank")}>Join our Discord!</Button>
```
Comment (will-nemo) [inline src/pages/ProjectsView.jsx:24]: no ! in Join our Discord

**Pull request merged**